faebryk/library/library/footprints.py

```diff
@@ -26,6 +26,22 @@ def get_kicad_footprint() -> str:
 
         self.add_trait(_has_kicad_footprint())
 
+class QFN(Footprint):
+    def __init__(self, pin_cnt: int, size_x_mm: int, size_y_mm: int) -> None:
```

Comment:
Add
- exposed thermal pad count
- lead/pin pitch
- exposed thermal pad size
- yes/no thermal vias in exposed thermal pad (only if thermal pad is >=1